pkg/sql/app_stats.go

```diff
@@ -287,7 +287,10 @@ func (e *Executor) GetScrubbedStmtStats() []roachpb.CollectedStatementStatistics
 			// guesstimate that we'll need apps*(queries-per-app).
 			ret = make([]roachpb.CollectedStatementStatistics, 0, len(a.stmts)*len(e.sqlStats.apps))
 		}
-		hashedApp := HashForReporting(salt, appName)
+		hashedApp, err := HashForReporting(salt, appName)
+		if err != nil {
+			hashedApp = "unknown"
```

Comment:
does this actually happen? assuming no, can we just panic on empty?

Reply:
(whoops, meant to put this comment on the == "" check below)

Reply:
It shouldn't.  I'll make the change.

Reply:
Eh, actually, I've convinced myself you were right here -- it should never happen, but if it does, I'd rather get junk reporting -- which just annoys us -- than crash the node for a non-critical process.

Reply:
Ok.... should I revert the change I just made?

Reply:
How about I put the "unknown" in the function as the return value and not return an error?

Reply:
👍 looks fine as-is, or that way. it's _green_ now though

Reply:
I'd rather get it right.  I'm going to be unknown in instead of the panic.